main.py

```diff
@@ -44,34 +44,64 @@ def read_pokemon(
     sort_by: str = Query(default="pokemon_id", description="Column to sort by", regex="^(pokemon_id)$"),
```

Comment:
try like this

```python
from typing import Literal

def get_all_pokemon(
    sort_by: Literal[<all accepted fields>] = Query(default="pokemon_id",  description="xxx"  ),
    sort_order: Literal["asc", "desc"] = Query("asc", description="xxx"  )
)
``` 

This will accept only the elements(strings here) mentioned in the list as inputs.